engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -204,6 +206,22 @@ public Vector3f getViewingDirection() {
         return viewingDirection;
     }
 
+    /**
+     * Get the gaze direction.
+     * @return the gaze direction, a quaternion.
+     */
+    public Quat4f getGazeDirection() {
```

Comment:
Hmmm... here I'm concerned the camera is getting a bit "too smart". 

I appreciate your VR camera vs Normal Camera clarification: creative and to the point. But I'm thinking: the camera shouldn't argue if it can or cannot do something. The camera should  be pretty dumb and do what it is told. Systems perhaps could fight each other to determine who has control. Or perhaps the LocalPlayerSystem should be more VR-aware than it is now.

I mean: remember that one of the goal we have about cameras is that they should become even dumber components. So, if you add this functionality, we'll then have to strip it away and put it back into a system. 

I'd therefore like to ask you if you can go the extra-mile and handle these issues at system level rather than camera level. Who controls the camera should not be relevant for the camera.